stl/inc/algorithm

```diff
@@ -394,6 +793,194 @@ _NODISCARD pair<_FwdIt1, _FwdIt2> mismatch(
 }
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    // ALIAS TEMPLATE mismatch_result
+    template <class _In, class _Out>
+    using mismatch_result = in_in_result<_In, _Out>;

```

Comment:
There's no thing "out" in `mismatch`.

Reply:
`in_in_result<_In, _Out>` - this must have been copy-pasta. I can't imagine that I deliberately chose the placeholder `_Out` for the type of `in2`.